### ISSUE TITLE
Pokétwo Has Leaked its mongodb IP Address in #flex of the official server.

### ISSUE THREAD

**Issue by Drxxy77** [poketwo/poketwo#174]
In the #flex channel this message was sent by poketwo:
204.16.243.193:27017: [Errno 111] Connection refused,204.16.243.194:27017: [Errno 111] Connection refused,mongodb-arbiter-0.mongodb-arbiter-headless.poketwo.svc.cluster.local:27017: [Errno -2] Name or service not known, Timeout: 30s, Topology Description: <TopologyDescription id: 6050bacfc7f81fcec27d48e0, topology_type: ReplicaSetNoPrimary, servers: [<ServerDescription ('204.16.243.193', 27017) server_type: Unknown, rtt: None, error=AutoReconnect('204.16.243.193:27017: [Errno 111] Connection refused')>, <ServerDescription ('204.16.243.194', 27017) server_type: Unknown, rtt: None, error=AutoReconnect('204.16.243.194:27017: [Errno 111] Connection refused')>, <ServerDescription ('mongodb-arbiter-0.mongodb-arbiter-headless.poketwo.svc.cluster.local', 27017) server_type: Unknown, rtt: None, error=AutoReconnect('mongodb-arbiter-0.mongodb-arbiter-headless.poketwo.svc.cluster.local:27017: [Errno -2] Name or service not known')>]>

As you can see:
204.16.243.194:27017  `//27017 is the default mongodb port`
This is the ip & port used for poketwo's mongodb, i suggest you fix this and get it changed :p

-Love KeemROH

**Comment by SVGEMS** [comment]
I got this too. I had my channels locked so no commands were being used, just randomly spit that info out.

**Comment by NathNathNathNath** [comment]
I doubt this is an issue - I'm sure the db has appropriate auth, etc
This bug should probably be reworked to "ensure  **only** end-user-friendly error messages are given", instead of being focused on the ip leak. imo anyway

**Comment by Drxxy77** [comment]
> I doubt this is an issue - I'm sure the db has appropriate auth, etc
> This bug should probably be reworked to "ensure **only** end-user-friendly error messages are given", instead of being focused on the ip leak. imo anyway

I'm sure that there are malicious people out there who have enough understanding to know that "Hey this is poketwo's database, im going to DDoS it."

**Comment by NathNathNathNath** [comment]
Yeah I get that, I'm just saying that the leak itself isn't a bug to be fixed. The bug to be fixed is the _cause_ of the leak - which is the error message being returned

**Comment by Drxxy77** [comment]
> Yeah I get that, I'm just saying that the leak itself isn't a bug to be fixed. The bug to be fixed is the _cause_ of the leak - which is the error message being returned

Oh yeah, ofc
